WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -758,8 +759,10 @@ nick_of_a_thousand_faces = {
 		age = 16
 		intrigue = 10
 		trait = disfigured
-		trait = deceitful
-		trait = envious
+		OR = {
+			trait = deceitful
+			trait = envious
+		}
```

Comment:
It's a mistake. 

A person who is Deceitful and Disfigured could be said to be of many faces.
A person who is Envious and Disfigured could be said to be of many faces.

Either combination would accurately portray the Nickname. Nickname triggers exist to ensure that people that match the name get it. To require both is absurd.